index.html

```diff
@@ -31,4 +31,4 @@
       Read more
```

Comment:
Your code works great! 
However, the content, 'Read more' of the `<a>` element should be on the same line as the opening and closing tags.

`<a href="https://en.wikipedia.org/wiki/Albert_Einstein" target="_blank" id="tribute-link"> Read more </a>`

This makes your code cleaner and more readable 😃